crates/sui/tests/reconfiguration_tests.rs

```diff
@@ -1251,10 +1273,11 @@ async fn execute_join_committee_txes(
 async fn execute_leave_committee_tx(
     authorities: &[SuiNodeHandle],
     gas: ObjectRef,
-    node_config: &NodeConfig,
+    account_kp: &Ed25519KeyPair,
```

Comment:
We shouldn't assume EdDSA keys, but this is a test, so we are good?

Reply:
yes this is fine in test. The default is ed25519